test/sdk/data-api.ts

```diff
@@ -465,12 +502,30 @@ describe("DataApi Class", async () => {
         }
       });
     });
+
+    describe("with binary / base64 field data", async () => {
+      it("encodes binaryFields data", async () => {
```

Comment:
maybe a test for verifying that a binary field can be cleared should also be added?

Reply:
`VersionData` is required, so there's no scenario where a dev would clear the field upstream. I did add a test to ensure that if `binaryFields` wasn't provided or was just an empty object, that we could still update the other fields. It's a good thing I added the test, because it did uncover a potential bug!